src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -95,9 +95,16 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + feedbackResponseComment.feedbackSessionName + "<br>"
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
-        }
 
-        data.comment = feedbackResponseComment;
+            String commentGiverName = logic.getInstructorForEmail(courseId, frc.giverEmail).name;
+            String commentEditorName = instructor.name;
+
+            // createdAt and lastEditedAt fields in updatedComment as well as sessionTimeZone
+            // are required to generate timestamps in editedCommentDetails
+            data.comment = updatedComment;
+            data.sessionTimeZone = session.getTimeZone();
+            data.editedCommentDetails = data.createEditedCommentDetails(commentGiverName, commentEditorName);
```

Comment:
Add a blank line before `data.editedCommentDetails = ...` too, otherwise it'll look like it's only part of this block and has no relation to the above block containing `commentGiverName` and `commentEditorName`.